test/dense/linearsystemsolver.jl

```diff
@@ -24,11 +24,12 @@ import LinearAlgebra.LAPACK: gels!, gesv!, getrs!, getri!
 
     # if S is approximately equal to s, we defined then it's alright
     for i in 1:length(result)
-        @test result[i] ≈ right_answer[i]
+        @test Array(result[i]) ≈ right_answer[i]
```

Comment:
why we are converting this to `Array`?

Reply:
Because the ≈ operator  to compare a CuArray and an Array is not implemented so I need to cast the `result` for GPU case.